.github/workflows/sweep-experiments.yml

```diff
@@ -0,0 +1,67 @@
+# Executes `gulp sweep-experiments` every monday.
+# If experiments are swept, a PR is created.
+
+on:
+  schedule:
+    - cron: '0 0 * * 1'
+
+jobs:
+  sweep-experiments:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: setup-node
+        uses: actions/setup-node@v2.1.2
+
+      - name: sweep
+        id: sweep
+        run: |
+          npm ci
+                    
+          git config --global user.name "${GITHUB_ACTOR}"
+          git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+
+          npx gulp sweep-experiments
```

Comment:
I suggest installing `gulp-cli` in a previous step and directly invoking the task, similar to this:

https://github.com/ampproject/amphtml/blob/6a609c6380bd7e052afe52b1c3e928d81694a529/.github/workflows/continuous-integration-workflow.yml#L23-L24

Additional totally-optional nit: Seems like the logs are losing their color, formatting, etc. On Travis, we use `unbuffer` to retain log formatting. Not sure if that's easily doable here. (I can try this later during the next cleanup)

https://github.com/ampproject/amphtml/blob/6a609c6380bd7e052afe52b1c3e928d81694a529/.travis.yml#L49-L54

Reply:
Probably something like:

```
unbuffer node `which gulp` sweep-experiments`
```

Should I try it out?

Reply:
On my machine, `which gulp` returns `/Users/rsimha/.nvm/versions/node/v14.15.1/bin/gulp` (the CLI executable, which redirects to whatever is inside `node_modules`), so I'm not sure it can be chained with a `node` command.

A reliable way I'm aware of is to chain `node_modules/gulp/bin/gulp.js` with the `node` invocation, like so:

```sh
unbuffer node node_modules/gulp/bin/gulp.js sweep-experiments
```

Feel free to ignore if it takes more than a quick attempt.

Reply:
Tried it on my personal fork, `unbuffer` doesn't seem to work at all :( 

```
/home/runner/work/_temp/...sh: line 4: unbuffer: command not found
```

Reply:
On Travis, we do this: 

https://github.com/ampproject/amphtml/blob/43a209eaa3c2919ca674e32be0ef1d6bd62eee24/.travis.yml#L26-L30

Not worth fighting the fight here :)